spot_driver/launch/spot_driver.launch.py

```diff
@@ -106,6 +109,14 @@ def launch_setup(context: LaunchContext, ld: LaunchDescription) -> None:
 
     ld.add_action(rviz)
 
+    depth_images = IncludeLaunchDescription(
+        PythonLaunchDescriptionSource(
+            [PathJoinSubstitution([FindPackageShare(THIS_PACKAGE), "launch", "spot_depth_publishers.launch.py"])]
```

Comment:
Make sure you're consistent about how you refer to these nodelets, since it's important that they're specifically publishing registered depth images. I'd suggest using more detailed names for the launch file and the `depth_images` variable here.